layouts/index.html

```diff
@@ -54,9 +60,14 @@ <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
         var button = document.getElementById( 'js-user-submit' );
         button.addEventListener( 'click', function ( event ) {
           event.preventDefault();
+          {{ if eq $.Site.Params.language "english" }}
+          var registerPath = "register/";
```

Comment:
Is there a way to pull the `register`/`registrar` string from a config variable or something? (To avoid an `if` here)